perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -206,8 +212,25 @@ public static String join(Iterable<?> collection, String separator) {
 							throw new ConsistencyErrorException("Creating existing extSource: " + additionalExtSourceName);
 						}
 					}
+					UserExtSource userExtSource = new UserExtSource(additionalExtSource, additionalExtLoa, additionalExtLogin);
+
+					List<Attribute> attributes = new ArrayList<>();
+					// Set ues attribute
+					if (additionalExtLoginAndAttribute.length == 2) {
```

Comment:
There could be more than 1 attribute after the login. You should parse all of them. So this code should probably work for length>1.